### ISSUE TITLE
Tornado incompatibility

### ISSUE THREAD

**Issue by trilbytim** [nikhil-sethi/xflrpy#3]
pip installing this causes it to downgrade Tornado to version 4, which is now 5 years old. This breaks a lot of dependencies on my system (including Jupyter, which is my preferred development environment). So it could really do with updating to be able to use a recent version of Tornado.

**Comment by trilbytim** [comment]
So it seems to be using msgpack-rpc-python which is problem, this is an old package, not updated since 2018. It appears to have been replaced by rpc-msgpack, which was last updated earlier this year, so is presumably current.
https://pypi.org/search/?q=msgpack-rpc-python
I don't know what purpose this rpc-msgpack is performing for XFLRpy, but would it be possible to switch to using a current version?

**Comment by nikhil-sethi** [comment]
Hey @trilbytim. I am aware of this issue and I have traditionally solved it using virtual environments but you're right that the current library is indeed outdated.
I have updated the requirements and client side to use rpc-msgpack.
As a side note, rpc-msgpack is being used to call existing C++ functions from a python API. It is a like a middleware for communication between the two.
Closing the issue. Feel free to open if you still face problems.
